ui/components/Navigator.js

```diff
@@ -473,6 +473,15 @@ const categories = [
       },
     ],
   },
+  {
+    id : "Extension",
+    // icon: <ConformanceIcon style={drawerIconsStyle} />,
+    // hovericon : <ConformanceHover style={drawerIconsStyle} />,
```

Comment:
We need an icon here to show in the navigator.

Reply:
here we are - ![meshery-extensions](https://user-images.githubusercontent.com/7570704/184353242-b6c3c520-3f1c-4cce-a67f-a4afa4962ef8.svg)